fe/src/App.tsx

```diff
@@ -2,16 +2,20 @@ import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import Home from './pages/Home/index';
 import About from './pages/About/index';
 import Error from './pages/Error/index';
```

Comment:
I know this is not from this pr but these switched back to `/index` at some point ):
@tomimarkus991 I suspect it's you.

@GeorgDV Please fix

Reply:
Oh right.. sure.